guide/onboarding/manual-backup.md

```diff
@@ -11,9 +11,30 @@ main_classes: -no-top-padding
 
 # Manual backup
 
-### How it works
+## Backing up channel state
 
-It usually requires users to manually back up their 12 to 24 word recovery phrase by writing it down on a piece of paper and storing it in a safe (but memorable location). In the case that a user’s device breaks or is stolen, the user can recover their funds and wallet by correctly entering their recovery phrase. The [private key management]({{ '/guide/private-key-management/manual-backup/' | relative_url }}) section dives further into the technical details of this scheme if interested in learning more.
+A channel state backup is typically a file which includes information about the user's channel balances. This file can be downloaded onto the user's device and used to restore the Lightning channel state on another device. This could be a database of the most current channel balance information, or a static backup (as an absolute last resort).
+
+{% include tip/open.html color="red" icon="forbid" label="Manual channel state backups are for advanced users" %}
+
+Manual backups of Lightning channel state should only be performed with careful consideration. This is because the channel state backup may become outdated very quickly, and attempting to restore from an outdated backup could harm the user's funds. In the interest of adhering to the principles of [self-custody]({{ '/guide/designing-products/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/designing-products/principles/#transparency' | relative_url }}), you may choose to allow your user to perform a manual channel state backup. However, this is not recommended for the majority of users. If this option is available, keep it tucked away in a settings menu and prepend a warning that the option is only for advanced users.
+
+{% include tip/close.html %}
+
+## Backing up wallet recovery data
```

Comment:
Could make a note in this section about offering the option to print out a recovery phrase template and (maybe?) that comes with a QR code of the wallet descriptor? We should make this a resource and link to it in `Design resources` in DBP.

Reply:
I added a printable template by repurposing some assets from the Bitcoin Backup Helper hackathon project.